src/it/scala/org/zalando/spearheads/innkeeper/dao/PathsPostgresRepoSpec.scala

```diff
@@ -1,12 +1,15 @@
 package org.zalando.spearheads.innkeeper.dao
 
 import java.time.LocalDateTime
+
```

Comment:
why are these spaces in between imports added? I'm always trying to remove them? :)